### DISCUSSION TITLE
Lack of explanation between Semaphores and electric signals

### DISCUSSION THREAD

**Discussion by andbruu** [voxeltycoon/issues#371] [UI&UX]
Having only semaphores or both (+1 electric signals) available from start is part of the difficulty setting, but one setting allows me to build the cheap option (semaphore), and the other option allows me to build both.
I conclude that the option that allows both must be the easier difficulty option, but I do not know why.

This have left me to believe that the electric signal must have an advantages over the semaphore, as it normally require research and is more expensive to construct.
I would love that the tooltip makes it clear what advantages the electric signals provide.

**Comment by victorleblais** [comment]
I'd argue that the only advantage of signals with lights is that they're easier to understand at a glance. But they work exactly the same.

**Reply by hairintd** [reply]
At this time it can be easily implemented as removal of pre-signals in semaphor-version. 
Or may be add an additional speed limit for semaphore-controlled blocks

**Reply by Road-Train** [reply]
I wouldn't remove pre-semaphores as semaphores are an essential QoL tool for signalling junctions. Instead of a speed limit however I suggest a reaction delay for semaphores. Whereas lights would be instant, a semaphore has to change it's position which takes time, causing trains to stay longer than normal at signals. Wouldn't be much of a difficulty increase but it would make sense.
